modules/lti/src/main/resources/tools/courses/js/lti.recordings.js

```diff
@@ -75,6 +75,54 @@ function xhr(params, cb, fail, always) {
   request.send(params.data);
 }
 
+var courseID = $.getURLParameter("sid"),
+    user = [],
+    seriesTitle = undefined,
+    userURL = "/info/me.json",
+    seriesURL = '/api/series/'+ courseID + '/metadata';
+
+xhr({url: userURL, responseType: 'json'}, 
+  function(response) {
+    user = response.user;
+});
+
+xhr({url: seriesURL, responseType: 'json'}, 
+  function(response) {
+    seriesTitle = response[0].fields[0].value;
+});
+
+$('#feedbackBtn').on('click', function() {
+  var parts = [],
+      feedbackUrl = "https://docs.google.com/forms/d/e/1FAIpQLSeXmOzYY3rwuB3Plj27kMcI-8B7PpBHkZOo_zi8dd15Zv3u4Q/viewform"; 
+
+  var tempVars = {
+    user: user,
+    mediaPackage_series: seriesTitle,
+    mediaPackage_seriesid: 'S['+courseID+']',
+    mediaPackage_date: information.mediapackage.start
+  };  
+
+  if (tempVars.user['email']) {
+    parts.push('entry.508626498=' + tempVars.user.email);
+  }
+  if (tempVars.user['username']) {
+    parts.push('entry.277413167=' + tempVars.user.username);
+  }
+  if (tempVars.mediaPackage_series) {
+    parts.push('entry.1631189828=' + tempVars.mediaPackage_series);
+  }
+  if (tempVars.mediaPackage_seriesid) {
+    parts.push('entry.1638000924=' + tempVars.mediaPackage_seriesid);
+  }
+  if (tempVars.mediaPackage_date) {
```

Comment:
The mediapackage date is not really relevant because this feedback is essentially for the entire series and not for s specific recording so the date field will be incorrect. So this should not be filled in or sent to the google form.

Reply:
Changes pushed